src/oas2/transformers/getExamplesFromSchema.ts

```diff
@@ -0,0 +1,16 @@
+import { isObject } from 'lodash';
+import { Schema } from 'swagger-schema-official';
+
+export function getExamplesFromSchema(data: Schema) {
+  return isObject(data)
+    ? 'x-examples' in data
+      ? isObject(data['x-examples'])
+        ? data['x-examples']
+        : void 0
+      : 'example' in data
+      ? { default: data.example }
+      : 'x-example' in data
+      ? { default: data['x-example'] }
```

Comment:
If OAS2 supports example, we do not need to check for x-example. We just need to backfill "unsupported" parts.